modules/gitops-enable-traffic-management-and-metric-plugins-in-argo-rollouts.adoc

```diff
@@ -0,0 +1,91 @@
+
+To enable traffic management and metric plugins in Argo Rollouts, complete the following steps.
+
+.Procedure 
+
+. Log in to the {OCP} web console as a cluster administrator. 
+
+. In the *Administrator* perspective, click *Operators* -> *Installed Operators*.
+
+. Create or select the project where you want to create and configure a `RolloutManager` custom resource (CR) from the *Project* drop-down menu.
+
+. Select *{gitops-title}* from the installed operators.
+
+. In the *Details* tab, under the *Provided APIs* section, click *Create instance* in the *RolloutManager* pane.
+
+. On the *Create RolloutManager* page, select the *YAML view* and edit the YAML.
++
+.Example Adding the traffic management and metric plugins configuration in the `RolloutManager` CR
+[source,yaml]
+----
+apiVersion: argoproj.io/v1alpha1
+kind: RolloutManager
+metadata:
+  name: argo-rollouts
+spec:
+  plugins:
+    trafficManagement:
+      - name: argoproj-labs/gatewayAPI #<1>
+        location: https://github.com/sample-metric-plugin #<2>
+    metric:
+      - name: argoproj-labs/sample-prometheus #<3>
+        location: https://github.com/sample-trafficrouter-plugin #<4>
+        sha256: dac10cbf57633c9832a17f8c27d2ca34aa97dd3d #<5>
+----
+<1> Specifies the name of the trafficManagement plugin.
+<2> Specifies the location of the trafficManagement plugin.
+<3> Specifies the name of the metric plugin.
+<4> Specifies the location of the metric plugin.
+<5> Optional: Specifies the SHA256 signature of the plugin binary, which is downloaded and installed by the Rollouts controller.
+
+. Click *Create*.
+
+. In the *RolloutManager* tab, under the *RolloutManagers* section, verify that the *Status* field of the RolloutManager instance shows as *Phase: Available*.
+
+. Verify that the traffic management and metric plugins are installed correctly by completing the following steps:
+.. In the *Administrator* perspective, click *Workloads* -> *ConfigMaps*.
+.. Click the *argo-rollouts* ConfigMap.
++
+As a result, the plugins defined in the `RolloutManager` CR are updated in the *argo-rollouts* ConfigMap.
++
+.Example Updated traffic management and metric plugins in the *argo-rollouts* ConfigMap 
+[source,yaml]
+----
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: argo-rollouts-config
+  namespace: argo-rollouts
+  labels:
+    app.kubernetes.io/component: argo-rollouts
+    app.kubernetes.io/name: argo-rollouts
+    app.kubernetes.io/part-of: argo-rollouts
+data:
+  trafficRouterPlugins: |
+       - name: argoproj-labs/openshift #<1>
+         location: https://github.com/sample-trafficrouter-plugin #<2>
+         sha256: "" #<3>
+  metricPlugins: |
+      - name: "argoproj-labs/sample-prometheus" #<4>
+        location: https://github.com/sample-metric-plugin #<5>
+        sha256: dac10cbf57633c9832a17f8c27d2ca34aa97dd3d #<6>
+      - name: argoproj-labs/gatewayAPI #<7>
+        location: https://github.com/default-sample-trafficrouter-plugin #<8>
```

Comment:
```suggestion
data:
  metricPlugins: |
    - name: argoproj-labs/sample-prometheus
      location: https://github.com/sample-trafficrouter-plugin
      sha256: dac10cbf57633c9832a17f8c27d2ca34aa97dd3d
  trafficRouterPlugins: |
    - name: argoproj-labs/gatewayAPI
      location: https://github.com/sample-metric-plugin
      sha256: ""
    - name: argoproj-labs/openshift
      location: file:/plugins/rollouts-trafficrouter-openshift/openshift-route-plugin
      sha256: ""
```
Please change the numbering with description accordingly

Reply:
Fixed.

Reply:
```
  metricPlugins: |
    - name: argoproj-labs/sample-prometheus
      location: https://github.com/sample-trafficrouter-plugin
      sha256: dac10cbf57633c9832a17f8c27d2ca34aa97dd3d
  trafficRouterPlugins: |
    - name: argoproj-labs/gatewayAPI
      location: https://github.com/sample-metric-plugin
      sha256: ""
    - name: argoproj-labs/openshift
      location: file:/plugins/rollouts-trafficrouter-openshift/openshift-route-plugin
      sha256: ""
```